src/olympia/files/migrations/0017_auto_20211128_0808.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.9 on 2021-11-28 08:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('files', '0016_auto_20210929_1203'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='file',
+            name='is_webextension',
+        ),
+        migrations.AlterField(
+            model_name='fileupload',
+            name='source',
+            field=models.PositiveSmallIntegerField(choices=[(1, 'Developer Hub'), (2, 'Signing API'), (3, 'Addon API')], default=None, null=True),
+        ),
```

Comment:
Please remove that part as it's not related (we'll need to have it, but it doesn't cause any changes to the database and can be done separately)

Reply:
Hi @diox removed the above operation in the new commit.